docs/docs/extensions/compose/_category_.json

```diff
@@ -1,6 +1,6 @@
 {
   "label": "Jetpack Compose",
-  "position": 1,
+  "position": 2,
```

Comment:
I guess these files are generated but why are both this and the fullscreen docs marked as position 2?

Reply:
It's because I made changes to add the Accessibility and Fullscreen docs at the same time, but put the PRs up separately since they technically don't depend on each other.
In the end, it's going to be ordered alphabetically so `0: Accessibility`, `1: Fullscreenshot`, `2: Jetpack Compose`